gh-upload

```diff
@@ -90,11 +98,19 @@ fi
 files=()
 blobs=()
 for file in "$@"; do
-    sha=$(gh api /repos/$owner/$repo/git/blobs -f "encoding=base64" -f "content=$(base64 $file)" | jq -r '.sha')
+    if [[ $(uname) == 'Darwin' ]]; then
+        content=$(base64 -i $file)
+    else
+        content=$(base64 $file)
+    fi
+    sha=$(gh api /repos/$owner/$repo/git/blobs -f "encoding=base64" -f "content=$content" | jq -r '.sha')
```

Comment:
```suggestion
content=$(base64 -i $file)
sha=$(gh api /repos/$owner/$repo/git/blobs -f "encoding=base64" -f "content=$content" | jq -r '.sha')
```

I don't think this condition is necessary, as base64 supports the -i option on both Mac and Linux platforms.